hack/build-cross.sh

```diff
@@ -12,6 +12,11 @@ if [[ -n "${OS_ONLY_BUILD_PLATFORMS-}" ]]; then
       filtered+=("${platform}")
     fi
   done
+  if [[ ${OS_ONLY_BUILD_PLATFORMS} == "linux/ppc64le" ]]; then
```

Comment:
I was debating about that.  But the problem would then be that ppc64le and aarch64 would be built each time someone did a cross-compile.
When building rpm's that do a cross compiled, we are already building for linux/i386, which we then just throw away.  I was trying to not have 2 more long builds that get thrown away 99% of the time.